src/sysv-generator/sysv-generator.c

```diff
@@ -568,7 +568,7 @@ static int load_sysv(SysvStub *s) {
                                                 }
                                         }
 
-                                        if (r < 0)
+                                        if (r < 0) // lgtm[cpp/constant-comparison]
```

Comment:
Were these changes cherry picked from somewhere? If yes, please mention the patch in the commit message. If not, please add "rhel-only" at the end of the commit message.

Also, use the c-style comment, as mentioned in the other patch.